xlml/apis/test_config.py

```diff
@@ -215,6 +232,41 @@ def test_script(self) -> str:
     return '\n'.join(('set -xue', *self.run_model_cmds))
 
 
+@attrs.define
+class CpuGkeTest(TestConfig[Cpu]):
```

Comment:
Nit: `Test config that runs on a single Cloud TPU instance in GKE cluster.` --> update TPU to CPU.

Also, please check the other TPU in this comment.

Reply:
Done - thank you!